consensus/polybft/extra_test.go

```diff
@@ -369,46 +369,52 @@ func TestExtra_CreateValidatorSetDelta_Cases(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		t.Run(c.name, func(t *testing.T) {
+		localCase := c
```

Comment:
Is this an issue because the tests run in Parallel?

Reply:
Yes and I fixed it because VSC reported violation (although in this case we have only single test case to run). However without having `t.Parallel()` I'd say sub-tests would be executed in parallel because of `t.Run` which spawns a routine.

Reply:
@ferranbt more detailed explanation [here](https://github.com/0xPolygon/polygon-edge/pull/1053#issuecomment-1353257905)